README.md

```diff
@@ -1 +1,237 @@
 # Gadget_Inspection_Template
+
+The gadget is made up of several docker containers. To run it you will need to have docker and docker-compose installed on your machine.
+
+The containers definition and configuration is done in a docker-compose yaml file, by default named inspection.docker-compose.yaml. In that file you'll find all the containers that make up the gadget. For organization sake they are put into three categories. Platform services make up the underlying components of the gadget that are consistent though out applications. Inspection services are configurable for a specific application and do the work of actually implementing the inspection. GoFactory services allow connection to the a GoFactory instance either running in the cloud on prem.
```

Comment:
``
GoFactory services allow connection to the a GoFactory instance either running in the cloud on prem.
``
is it missing a word `` or ``?